### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by swapnanildutta** [swapnanildutta/CoronavirusDatabase#3]

**Comment by VishalAmbavade** [comment]
Hi @swapnanildutta, can you provide more information regarding your issue please?

**Comment by swapnanildutta** [comment]
Yeah sure. In the README provide which file does what and how to execute them as well as graphical representation if possible.

**Comment by abhishekpatel946** [comment]
Hey, @swapnanildutta 
can you assign to me I want to create an awesome README.md file

**Comment by swapnanildutta** [comment]
Yeah sure, you can create. You're most welcome.

**Comment by Nova-Striker** [comment]
@swapnanildutta @abhishekpatel946  , For Windows we have to activate the venv by `coronavirus_venv\Scripts\activate.bat`  and not by `coronavirus_venv\Scripts\activate`
